sql-plugin/src/main/scala/com/nvidia/spark/rapids/GpuOverrides.scala

```diff
@@ -1141,15 +1141,6 @@ object GpuOverrides extends Logging {
         override def convertToGpu(child: Expression): GpuExpression =
           GpuUnaryMinus(child, ansiEnabled)
       }),
-    expr[UnaryPositive](
-      "A numeric value with a + in front of it",
-      ExprChecks.unaryProjectAndAstInputMatchesOutput(
-        TypeSig.astTypes + GpuTypeShims.additionalArithmeticSupportedTypes,
-        TypeSig.gpuNumeric + GpuTypeShims.additionalArithmeticSupportedTypes,
-        TypeSig.numericAndInterval),
-      (a, conf, p, r) => new UnaryAstExprMeta[UnaryPositive](a, conf, p, r) {
```

Comment:
Personally i would use `ExprMeta` here instead of `UnaryAstExprMeta` to fix the build error. Then no shim is needed. 
But i am ok with the fix by shims.

Reply:
That's much better as it reduces the code churn. I wanted to keep it restricted to extend a unary expression but Spark has removed that restriction so it is not that fruitful. I will follow your suggestion and revert this